### PR TITLE
Implement PropertyReader and EnvironmentConfig

### PR DESCRIPTION
**Pull request by Astashevich**

### CODE REVIEW

File: src/main/resources/application-dev.properties [hunk 1]
```diff
@@ -0,0 +1,13 @@
+environment=DEV
+###########################################################
+# Web application properties
+###########################################################
+host=https://www.onliner-dev.by/
+###########################################################
+# Files path properties
+###########################################################
+logPath=target/logs/appTest.log
```
Comment (Anassstasi) [inline src/main/resources/application-dev.properties]: Are these two parameters different for all the envs or common?

File: src/main/resources/application-qa.properties [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# Web application properties
+###########################################################
+host=https://www.onliner-qa.by/
+###########################################################
+# Files path properties
+###########################################################
+logPath=target/logs/appTest.log
+saveVideoPath=./Videos/
+###########################################################
+# Test data
```
Comment (Anassstasi) [inline src/main/resources/application-qa.properties]: You can store valid log in and psw in property files

Comment (Astashevich) [reply src/main/resources/application-qa.properties]: OK

File: src/main/resources/application-dev.properties (hunk 1 above)
Comment (Astashevich) [reply src/main/resources/application-dev.properties]: Back this two parameters in to Constant

File: src/main/java/by/onliner/core/properties/EnvironmentConfig.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+/***
+ * Project environment data
+ */
+public class EnvironmentConfig {
+
+    private static final String HOST = "host";
+    private static final Properties properties;
+
+    private final static String ENVIRONMENT_PROPERTIES_FILENAME = String.format("application-%s.properties", Environment.getEnvironment().getName());
```
Comment (Anassstasi) [inline src/main/java/by/onliner/core/properties/EnvironmentConfig.java]: Why is this variable global? are you going to use it somewhere else?

Comment (Astashevich) [reply src/main/java/by/onliner/core/properties/EnvironmentConfig.java]: Fixed

**Review by Anassstasi — APPROVED** [review]

**Pull request merged**